e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/view-utilisation.spec.js

```diff
@@ -0,0 +1,122 @@
+      ],
+    };
+
+    cy.task(NODE_TASKS.INSERT_TFM_FACILITIES_INTO_DB, [tfmFacilityOne, tfmFacilityTwo]);
+
+    cy.task(NODE_TASKS.DELETE_ALL_FROM_SQL_DB);
+
+    const report = UtilisationReportEntityMockBuilder.forStatus(UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION)
+      .withId(REPORT_ID)
+      .withBankId(BANK_ID)
+      .build();
+    cy.task(NODE_TASKS.INSERT_UTILISATION_REPORTS_INTO_DB, [report]);
+
+    const feeRecordOne = FeeRecordEntityMockBuilder.forReport(report)
+      .withId(FEE_RECORD_ID_ONE)
+      .withFacilityId(FACILITY_ID_ONE)
+      .withExporter('Exporter 1')
+      .withBaseCurrency(CURRENCY.GBP)
+      .withFacilityUtilisation(500000)
+      .withTotalFeesAccruedForThePeriod(300)
+      .withTotalFeesAccruedForThePeriodCurrency(CURRENCY.EUR)
+      .withFeesPaidToUkefForThePeriod(100)
+      .withFeesPaidToUkefForThePeriodCurrency(CURRENCY.JPY)
+      .withPaymentExchangeRate(2)
+      .withPaymentCurrency(CURRENCY.USD)
+      .withStatus(FEE_RECORD_STATUS.TO_DO)
+      .build();
+
+    const feeRecordTwo = FeeRecordEntityMockBuilder.forReport(report)
+      .withId(FEE_RECORD_ID_TWO)
+      .withFacilityId(FACILITY_ID_TWO)
+      .withExporter('Exporter 2')
+      .withBaseCurrency(CURRENCY.GBP)
+      .withFacilityUtilisation(199999.99)
+      .withTotalFeesAccruedForThePeriod(200)
+      .withTotalFeesAccruedForThePeriodCurrency(CURRENCY.EUR)
+      .withFeesPaidToUkefForThePeriod(100)
+      .withFeesPaidToUkefForThePeriodCurrency(CURRENCY.GBP)
+      .withPaymentExchangeRate(2)
+      .withPaymentCurrency(CURRENCY.USD)
+      .withStatus(FEE_RECORD_STATUS.TO_DO)
+      .build();
+
+    cy.task(NODE_TASKS.INSERT_FEE_RECORDS_INTO_DB, [feeRecordOne, feeRecordTwo]);
+
+    pages.landingPage.visit();
+    cy.login(USERS.PDC_RECONCILE);
+
+    cy.visit(`utilisation-reports/${REPORT_ID}`);
+
+    pages.utilisationReportPage.utilisationTabLink().click();
+  });
+
+  it('should render the utilisation for each fee record', () => {
+    pages.utilisationReportPage.utilisationTab.utilisationTable.row(FEE_RECORD_ID_ONE).within(() => {
+      cy.assertText(cy.get('[data-cy="facility-id"]'), FACILITY_ID_ONE);
+      cy.assertText(cy.get('[data-cy="exporter"]'), 'Exporter 1');
+      cy.assertText(cy.get('[data-cy="base-currency"]'), CURRENCY.GBP);
+      cy.assertText(cy.get('[data-cy="value"]'), '1,000,000.00');
+      cy.assertText(cy.get('[data-cy="utilisation"]'), '500,000.00');
+      cy.assertText(cy.get('[data-cy="cover-percentage"]'), '80%');
+      // 80% of 500,000 is 400,000
+      cy.assertText(cy.get('[data-cy="exposure"]'), '400,000.00');
+      cy.assertText(cy.get('[data-cy="fees-accrued"]'), `${CURRENCY.EUR} 300.00`);
+      cy.assertText(cy.get('[data-cy="fees-payable"]'), `${CURRENCY.JPY} 100.00`);
+    });
+
+    pages.utilisationReportPage.utilisationTab.utilisationTable.row(FEE_RECORD_ID_TWO).within(() => {
+      cy.assertText(cy.get('[data-cy="facility-id"]'), FACILITY_ID_TWO);
```

Comment:
Could we move the `cy.get` selectors into a `pages` file please so these can become e.g

```js
cy.assertText(thePage.facilityId, FACILITY_ID_TWO);
```

If all the existing tests for utilisation reports have hard coded `cy.get`'s, not a blocker as would like it to be consistent.